spec.bs

```diff
@@ -1171,11 +1252,7 @@ To <dfn>convert a modifier to a string</dfn> given a [=part/modifier=] |modifier
   To <dfn>process pathname for init</dfn> given a string |pathnameValue|, a string |protocolValue|, and a string |type|:
 
   1. If |type| is "`pattern`" then return |pathnameValue|.
-  1. Let |dummyURL| be a new [=URL record=].
-  1. If |protocolValue| is null or the empty string, then set |protocolValue| to "`https`".
-  1. Set |dummyURL|'s [=url/scheme=] to |protocolValue|.
-     <p class="note">Note, we set the [=URL record=]'s [=url/scheme=] in order to check if it [=is special=].  If |protocolValue| is missing we purposely default the URL to being special by using "`https`" for the [=url/scheme=].</p>
-  1. If |dummyURL| [=is special=], then:
+  1. If |protocolValue| is a [=special scheme=], then:
```

Comment:
Should this be part of the previous PR?

Reply:
I could try to move it back, but I did it here since I figured out the better way to do the "is special" check.  If its ok, can I just leave it here instead of trying to move between the dependent branches?